src/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import * as PropTypes from "prop-types";
+
+import {Label} from "../Label";
+import {Button} from "../Button";
+import {FormGroupContextTypes, FormGroupContext} from "../FormGroup";
+
+import {RadioGroupProps} from "./RadioGroupProps";
+
+export class RadioGroup extends React.Component<RadioGroupProps, undefined> {
```

Comment:
This component has two roles: view and controller.
There is should be component like Button with action prop (value, which will be passed to onChange).
```tsx
// should be in your project, not context-form
import * as React from "react";

const yourAwesomeComponent = () => (
    <div className="radio-group">
        <Label>Some text</Label>
        <Button action={true}/> 
        <Label>Not text</Label>
        <Button action={false}/>
    </div>
);

```